project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -224,6 +224,28 @@ function c4m_og_menu() {
 function c4m_og_menu_alter(&$items) {
   // Adding a member should be allow to SA only (CFM-1292).
   $items['group/%/%/admin/people/add-user']['access callback'] = 'c4m_og_add_member_to_group_access';
+
+  // Assign access to Node Gallery API features, if user has permission to
+  // edit the gallery.
+  $node_galery_api_features = array('upload', 'items', 'manage', 'sort');
+  foreach ($node_galery_api_features as $suffix) {
+    $path = 'node/%node_gallery_api_gallery/' . $suffix;
```

Comment:
let's drop the `$path` and use it directly in the `$items` variable (more readable for `hook_menu_alter`).